app/komponenter/feilside/Feilside.tsx

```diff
@@ -4,10 +4,15 @@ import Banner from '../banner/Banner';
 import HovedInnholdCss from '../hovedInnhold/hovedInnhold.module.css';
 import css from './feilside.module.css';
 import { ELocaleType } from '~/typer/felles';
+import { DecoratorElements } from '@navikt/nav-dekoratoren-moduler/ssr';
 
-const Feilside: React.FC = () => {
+interface Props {
+  dekoratørFragmenter: DecoratorElements;
```

Comment:
Denne brukes ikke utenom å sende videre til Dokument. Kunne Dokument ha hentet den inn selv i stedet for å få den via via?

Reply:
God ide! Er det sånn at man burde ha en OutletContext utenfor Dokument, og at denne også kunne holdt på språk sånn at man ikke trenger å sende språk inn som prop på Dokument også?

Reply:
Smart med dekoratør fragmentene og slippe å proppe de! Har endret her ([50629ce](https://github.com/bekk/nav-familie-endringsmelding/pull/50/commits/50629ce42908738a5a5d9ef22ecce538669dbfac)) til at Dokument og Oppsett bruker useLoaderData. Når det kommer til feilsiden kan ikke den som er i root vise dekoratøren slik vi hadde gjort her tidligere da det kan være på grunn av feil i hentingen av denne at feilsiden skal vises. Vi har diskutert at for eksempel feilmeldingssiden her ikke har dekoratør og står på bokmål ettersom vi heller ikke kan bruke sanity tekster, men at man kan sette inn flere ErrorBoundary-er i feks index som tar for seg feil som skjer i de ulike routsene. Gjerne kom med innspill ☺️

Reply:
Når det gjelder OutletContext er vi litt usikre på hvordan dette ville fungert ettersom Outlet kjører elementene på de forskjellige routes-ene og Dokument wrapper rundt disse routes-ene. @daphneleebeek har du noen tanker om hvordan man kunne løst det på den måten?